src/collections/news/2023-05-17-Baby-Bliss-Bot-AI-Techniques.md

```diff
@@ -41,20 +44,36 @@ Guided by the article [How to Train StyleGAN2-ADA with Custom Dataset](https://t
 the 1,216 single Bliss symbols were used to train the network.
 Training was conducted over several runs for total training time of 62 hours.
 The following figure show an array of the final fake images of Bliss symbols
-generated by the GAN:
+generated by the trained model:
 
 <img src="/news/images/StyleGAN2-ADA-fakes.png" alt="array of fake Bliss
 symbol images as generated by StyleGAN2-ADA">
 
-As can be seen, the fake images have a superficial resemblance to real Bliss
-symbols and it appears that the GAN has detected and learned some patterns
-within the single Bliss symbols. At the present time it is unclear what
-patterns the GAN has learned exactly.  This is an issue for potential future
-explorations of GAN machine learning models.
+As can be seen, the generated images have a superficial resemblance to real
+Bliss symbols and it appears that the GAN has detected and learned some
+patterns within the single Bliss symbols. At the present time it is unclear
+what patterns the GAN has learned exactly.  This is an issue for potential
+future explorations of GAN machine learning models.
 
 ### StyleGAN3
 
-- Cindy to help here
+StyleGAN3 introduced improvements over StyleGAN2, primarily addressing the issue of
+aliasing in generated images. Aliasing becomes particularly noticeable when creating
+rotating images, resulting in a "glued" appearance of pixels that fail to rotate
+naturally. To overcome this problem, StyleGAN3 ensures that every layer of the synthesis
+network produces a continuous signal, enabling the transformation of details collectively.
+
+Guided by the README of [the StyleGAN3 PyTorch implementation](https://github.com/NVlabs/stylegan3)
+and utilizing the same dataset of 1,216 single Bliss symbols used to train the
+StyleGAN2-ADA model, training for StyleGAN3 was conducted over a span of 240 hours.
+The following figure shows arrays of final fake images generated by the trained model:
+
+<img src="/news/images/StyleGAN3-fakes.png" alt="array of fake Bliss
```

Comment:
I decided to add the markup so that the StyleGAN2-ADA images were enclosed in `figure` elements with `figcaption`s, and centred horizontally.  You might want to do the same.  It would be something like:

```
<figure style="text-align: center;">
<img src="/news/images/StyleGAN3-fakes.png" alt="array of fake Bliss
symbol images as generated by StyleGAN3>
<figcaption>Figure 3. Array of images generated by StyleGAN3</figcaption>
</figure>
```
I think it's figure 3 since it is the third image.  I will double check after the merge.